packages/st2/debian/rules

```diff
@@ -71,6 +76,12 @@ override_dh_virtualenv:
 	# NB! Use '--copies' else /opt/stackstorm/st2/bin/python is symlink to
 	# /usr/bin/pythonx rather than copy - and this breaks pack install
 	# which follows the path to calculate the location of the bin directory
+
+	# Fix for dual virtualenv installation to fix issue: https://github.com/pypa/virtualenv/issues/1873
+	/usr/bin/test "$(DEB_DISTRO)" == "focal" && \
+		pip uninstall -y virtualenv && \
+		dpkg --force-all --purge python3-virtualenv && \
+		pip install virtualenv || true
```

Comment:
Can we move this part from the st2 deb/rpm build time to the pre-packaged environment in the https://github.com/StackStorm/st2packaging-dockerfiles/blob/master/packagingbuild/focal/Dockerfile
 Dockerfile?

So the environment will be ideally in the expected state.

Reply:
I've removed the `python3-virtualenv` debian package from the docker build image.  This should avoid the duplicate installation.  PR create https://github.com/StackStorm/st2packaging-dockerfiles/pull/106